pkg/operator/ceph/csi/spec.go

```diff
@@ -178,9 +179,10 @@ var (
 )
 
 const (
-	KubeMinMajor                     = "1"
-	KubeMinVerForOIDCTokenProjection = "20"
-	kubeMaxVerForBeta1csiDriver      = "21"
+	KubeMinMajor                       = "1"
+	KubeMinVerForOIDCTokenProjection   = "20"
```

Comment:
good time to drop this as minimum supported version in 1.21 https://rook.github.io/docs/rook/v1.11/Getting-Started/Prerequisites/prerequisites/#minimum-version